connector/pom.xml

```diff
@@ -14,6 +14,18 @@
 
     <dependencies>
```

Comment:
To avoid version collisions, we moved in the sdk in this version to using azure bom
Please use azure bom instead of specifying version of every   <groupId>com.azure</groupId> 
dependency
https://github.com/Azure/azure-kusto-java/blob/master/data/pom.xml
https://github.com/Azure/azure-kusto-java/blob/master/ingest/pom.xml

`<dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>com.azure</groupId>
                <artifactId>azure-sdk-bom</artifactId>
                <version>${azure-bom-version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>`